campaigns/001-netperf-data-exchange/run.py

```diff
@@ -66,6 +58,20 @@ def main(x):
     x.exec('102-tcp-route-metrics-save alpha route-metrics-save:disabled')
     x.exec('102-tcp-route-metrics-save beta route-metrics-save:disabled')
 
+    # interface offloading
+    # usage: [host] command:"[command]"
+    # restoration has to be covered manually (see end of run)
+    x.exec('105-offloading alpha command:"tso off ufo off gso off gro off lro off"')
```

Comment:
mmh, the syntax differs from all other applets - not really consistent. A except a) or b):

- tso:[on | off], gro:[on | off], ...
- offloading:[on | off]

Mhhhh